bower.json

```diff
@@ -1,17 +1,10 @@
 {
   "name": "ember-frost-core",
   "dependencies": {
-    "ember": "2.11.2",
-    "ember-cli-shims": "0.1.3",
-    "ember-inflector": "^1.3.1",
-    "ember-mocha": "0.8.11",
+    "ember": "~2.12.1",
```

Comment:
To be clear, the move of `ember` from `bower.json` to npm as `ember-source` was supposed to happen as part of the 2.11.x upgrade. It is also correct that the path to the template compiler will need to be updated.